src/handwritten/task.py

```diff
@@ -441,37 +438,34 @@ def perform_strain_shunt_cal(
 
     def perform_bridge_shunt_cal(
             self, channel="", shunt_resistor_value=100000,
-            shunt_resistor_location=0, shunt_resistor_select=0,
-            shunt_resistor_source=0, bridge_resistance=120,
+            shunt_resistor_location=ShuntElementLocation.R1, shunt_resistor_select=ShuntCalSelect.A,
+            shunt_resistor_source=ShuntCalSource.BUILT_IN, bridge_resistance=120,
             skip_unsupported_channels=False):
```

Comment:
Please match the LabVIEW VI's default values: 100000, **R3**, A, DEFAULT, 120
![image](https://github.com/ni/nidaqmx-python/assets/17914399/068dec28-94d7-458e-a5ed-1ff8c55f2c92)